executor/aggregate.go

```diff
@@ -293,7 +294,7 @@ func (e *HashAggExec) initForParallelExec(ctx sessionctx.Context) {
 	finalConcurrency := sessionVars.HashAggFinalConcurrency()
 	partialConcurrency := sessionVars.HashAggPartialConcurrency()
 	e.isChildReturnEmpty = true
-	e.finalOutputCh = make(chan *AfFinalResult, finalConcurrency)
+	e.finalOutputCh = make(chan *AfFinalResult, finalConcurrency+partialConcurrency+1)
```

Comment:
Why change this?

Reply:
revert.

Reply:
Not only finalWorker will send data/error to the channel, but also fetchChildWorker/partialWorker will send errors to the channel.
If the buffer is not enough, the send will be blocked, and maybe lead dead lock.

Reply:
What's the `+1` for ?

Reply:
fetchChildData() also use the channel.